File: 3DVisualizationModule/angular-three/src/app/interface/medical-record.ts
export interface MedicalRecord {
  _id: string;
  domainId: string;
  patientId: string;
  allergies: string[];
  medicalConditions: string[];
  freeText: string;
  createdAt: string;
  updatedAt: string;
  __v: number;
}

File: backend-module/Models/MedicalRecord/MedicalRecord.cs
namespace backend_module.Models.MedicalRecords {
public class MedicalRecord
    {
        public string Id { get; set; }
        public string patientId { get; set; }
        public List<string> allergies { get; set; }
        public List<string> medicalConditions { get; set; }
        public string freeText { get; set; }
    }
}